### PR TITLE
LaTeX guidelines markdown

### PR DESCRIPTION
**Pull request by ezgimez**

### CODE REVIEW

File: LaTeX_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+For subsections, do:
+```tex
+\label{subsec:XX}
+```
+I hope you get the idea :)
+
+To refer to these labels, let's say a figure, in the text simply do:
+```tex
+\ref{fig:XX}
+```
```
Comment (lhl2617) [inline LaTeX_guidelines.md]: Would add a note on doing `Figure~\ref{fig:XX}`

Comment (ezgimez) [reply LaTeX_guidelines.md]: why ~ ?

Comment (lhl2617) [reply LaTeX_guidelines.md]: https://www.andy-roberts.net/writing/latex/floats_figures_captions

Comment (lhl2617) [reply LaTeX_guidelines.md]: For my projects sometimes I have an extra macro for this (\reffig), but sometimes you also want Fig.~\ref, so I lost use of that and I can't remember where I last used it

Comment (ezgimez) [reply LaTeX_guidelines.md]: I doubt that people will read this document and put the '~' after the 'Figure'... but fingers crossed -- revising now

**Review by lhl2617 — APPROVED** [review]
318CF1C9-58F6-46F3-9535-8B2554A5DCF8

Btw see comments on #3

**Comment by ezgimez** [comment]
folks, I'm gonna merge this PR (honesty, doesn't include any major stuff) if this doesn't get merged after Dec 28.
xxx

**Comment by ezgimez** [comment]
thanks boiii @NGovani

**Pull request merged**